include/glow/Graph/Graph.h

```diff
@@ -176,9 +176,11 @@ class Module final {
   /// Erase all of the functions from the module.
   void eraseFunctions();
 
-  /// Erase all the functions, variables, etc. If \p clearPlaceholders is false
-  /// then placeholders in the module will not be cleared out.
-  void clear(bool clearPlaceholders = true);
+  /// Erase all the functions, variables, etc.
+  void clear();
+
+  /// Strips payloads from constants and deletes functions.
```

Comment:
Can you add more explanations here, e.g. that it keeps the metadata around and why it does it? And may be explain for `clear` that it removes everything and how both are different.